container_pipeline/pipeline.py

```diff
@@ -75,6 +50,11 @@ def create_project(appid, jobid, repo_url, repo_branch, repo_build_path,
      target_file, notify_email, desired_tag, depends_on,
      test_tag) = sys.argv[1:]
     try:
-        create_project(*sys.argv[1:])
+        trigger_dockerfile_linter(appid, jobid, repo_url, repo_branch,
+                                  repo_build_path, target_file, notify_email,
+                                  desired_tag, depends_on, test_tag)
+        create_project(appid, jobid, repo_url, repo_branch, repo_build_path,
+                       target_file, notify_email, desired_tag, depends_on,
+                       test_tag)
     except OpenshiftError:
```

Comment:
LGTM, but could we not create the project only if the docker file is present for building. Though it does not matter as we are blocking the build, if linter is not passing.